docs/release_guide.md

```diff
@@ -34,12 +34,12 @@ Look for a branch on the upstream repo named `release-vX.Y`, where `X` and `Y` c
 
 Identify the base commit in the `main` branch for the release and cut a release branch off `main`.
 ```console
-$ git checkout -b release-<version> <commit-id> # ex: git checkout -b release-v0.4 0d05587
+$ git checkout -b release-<version> <commit-id> # ex: git checkout -b release-v0.9 0d05587
```

Comment:
I would argue against keeping this up to date since it doesn't seem any more helpful than what it was before. And we already have lots of places we need to keep the version up to date. Same goes for the other examples in this doc.

Reply:
Sounds good.